cumulus/client/cirrus-executor/src/bundle_processor.rs

```diff
@@ -1,7 +1,11 @@
-use crate::{aux_schema, ExecutionReceiptFor, SignedExecutionReceiptFor};
+use crate::{
```

Comment:
Out of curiosity, how long will we use Polkadot's code style here instead of default one we use everywhere else?

Reply:
Now it's totally fine by to remove the rustfmt.yml in cumulus folder and unignore `cumulus` from our custom rustfmt.yml: https://github.com/subspace/subspace/blob/aef122ef7794b34e0abca5a1772ef35b1453dc39/rustfmt.toml#L3-L4

It can be done in a separate PR if you want.